.github/workflows/stale.yaml

```diff
@@ -0,0 +1,23 @@
+name: 'Close stale issues and PRs'
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v8
+        with:
+          # Issues
+          stale-issue-message: 'This issue is stale because it has been open 180 days with no activity. Remove stale label or comment or this will be closed in 7 days.'
+          close-issue-message: 'This issue was closed because it has been stalled for 7 days with no activity. Feel free to reopen if still applicable.'
+          days-before-issue-stale: 180
+          days-before-issue-close: 7
+          exempt-issue-labels: 'kind/bug,good first issue'
```

Comment:
Should we add one more label that we can use to prevent the CI from deleting that issue manually? I mean, I think we should have a way to bypass the logic of removing issues automatically on purpose, wdyt?

Reply:
aka. I propose to add `help wanted` labels to exempt labels list.

Reply:
sounds good idea 👍

Reply:
that's better 🫡